.circleci/config.yml

```diff
@@ -6,31 +6,41 @@ orbs:
 executors:
   besu_executor_small:
     docker:
-      - image: cimg/openjdk:11.0
+      - image: cimg/openjdk:11.0.13
     resource_class: small
     working_directory: ~/project
     environment:
       GRADLE_OPTS: -Dorg.gradle.daemon=false -Dorg.gradle.parallel=true -Dorg.gradle.workers.max=2
 
   besu_executor_med: # 2cpu, 4G ram
     docker:
-      - image: cimg/openjdk:11.0
+      - image: cimg/openjdk:11.0.13
```

Comment:
> <openjdk-version> - The version of OpenJDK to use. This can be a full SemVer point release (such as 11.0.2) or just the minor release (such as 11.0). If you use the minor release tag, it will automatically point to future patch updates as they are released by AdoptOpenJDK & Oracle. \[https://circleci.com/developer/images/image/cimg/openjdk]

Do we really want to lock ourselves into a specific patch release?